coding-concepts.js

```diff
@@ -8,36 +8,46 @@
 // --------------------1) What will this log?
 
 const cohort = "Charlie 2023"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ['c', 'h', 'a', 'r', 'l', 'i', 'e', ' ', '2', '0', '2', '3']
+// b) Verify and explain: ['c', 'h', 'a', 'r', 'l', 'i', 'e', ' ', '2', '0', '2', '3'] was the output,
+// the method .split("") is used to split a string into an array of substrings based on a specified separator, which is the double quotes signs.
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:  nothing, it has no 'return' statement.
+// b) Verify and explain: My answer were partially correct, the output given was 'undefined' , because the function has no 'return' 
+// statement, which is used to specify a value that a function should return. When a 'return' statement is executed within a function, 
+// it immediately stops the execution of the function and returns the specified value, in this case it hasn't given any output because of that. 
 
 // --------------------3) What will this log?
 
-const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
+// const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
 // console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I't won't work because you are missing a { return } after the arrow function.
```

Comment:
When you write an arrow function like this without curly braces, the return is implied, which is why it will still work even though there is so `return` word actually in there.

Reply:
gotcha!